src/components/learner-credit-management/data/utils.js

```diff
@@ -134,3 +149,28 @@ export const formatPrice = (price) => {
   });
   return USDollar.format(Math.abs(price));
 };
+
+export const orderOffers = (offers) => {
```

Comment:
It may be helpful to add a JSdoc comment above this function to describe its intent/purpose in a sentence or two. Otherwise, I'm assuming the status order below indicates active then upcoming then expired, but may be worth describing in words too.

Reply:
Actually, I was working on this PR, and that's why this PR was in [WIP]. But now it is fixed.